app/jobs/searchgov_index_urls_job.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class SearchgovIndexURLsJob < ApplicationJob
+  queue_as :searchgov
+  delegate :upload_and_index, to: :@uploader
+
+  def perform(job_id, urls)
+    @time_started = Time.zone.now
+    @total_count = urls.count
+    @uploader = BulkUrlUploader.new(job_id, urls)
+
+    upload_and_index
+    log_results
+  end
+
+  def json_results
+    results = @uploader.results
+    {
+      job_id: results.name,
+      start_time: @time_started,
+      end_time: Time.zone.now,
+      indexed_count: results.total_count,
+      failed_count: results.error_count,
+      total_count: @total_count,
+      failed_urls: results.errors
+    }.to_json
+  end
+
+  def log_results
+    results = @uploader.results
+    Rails.logger.info "SearchgovIndexURLsJob: #{results.name}"
+    Rails.logger.info "    #{results.total_count} URLs"
```

Comment:
Let's keep logs info in one line because when we'll see this log by itself it won't make any sense. You can pass a message and include more information on it such as:
```
Rails.logger.info "SearchgovIndexURLsJob: #{results.name}", total_count: results.total_count, errors_count: results.error_count
```

Reply:
Good catch 🎣 

Completely forgot about this